src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -57,12 +57,19 @@ const Body = BaseForm.extend(Object.assign(
         : loc('mfa.calling', 'login');
       const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
+      let nicknameText = this.model.get('nickname') || '';
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
+      if (nicknameText !== '') {
+        nicknameText = '(' + nicknameText + ')';
+      }
 
       const strongClass = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
         'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">
-        ${sendText}&nbsp;<span class='${strongClass}'>${this.model.escape('phoneNumber')}.</span>
+        ${sendText}&nbsp;<span class='${strongClass}'>${this.model.escape('phoneNumber')}.${nicknameText}</span>
```

Comment:
nicknameText should only be added if this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login'), meaning the phoneNumber is like XXX-XXX-5455 instead of the text phone.

Reply:
Addressed.